src/react/ui-elements/__tests__/EmptyConfirmation.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { fireEvent, render } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { EmptyConfirmation } from '../EmptyConfirmation';
+
+const queryClient = new QueryClient();
+
+const bucketName = 'test-bucket';
+
+type WrapperProps = { children?: React.ReactNode };
+const Wrapper = ({ children }: WrapperProps) => (
+  <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+);
+jest.mock('../../next-architecture/ui/S3ClientProvider', () => ({
+  useS3Client: jest.fn(),
+}));
+jest.mock('react-redux', () => ({
+  useSelector: () => false,
+}));
+
+describe('EmptyConfirmation', () => {
```

Comment:
I think we are missing 3 cases:
- "should enable button once typed confirm"
- "should display count of deleted items while deleting objects"
- "should display the summary when the operation is done"